common/lib/dependabot/pull_request_creator/pr_name_prefixer.rb

```diff
@@ -421,6 +451,14 @@ def codecommit_client_for_source
           )
       end
 
+      def bitbucket_client_for_source
+        @bitbucket_client_for_source ||=
+          Dependabot::Clients::BitbucketTS.for_source(
```

Comment:
This looks like a typo maybe? Not seeing where BitbucketTS is defined.

Reply:
Indeed, BitbucketTS is the name I used to fork Bitbucket in the script we're using. Nice catch !